scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -44,149 +49,233 @@ set.seed(12345)
+
+
+```{r adt rows, live=TRUE}
 # What proteins were assayed?
-adt_tags <- rownames(altExp(sce))
+adt_tags <- rowData(altExp(sce))
 adt_tags
 ```
 
+You can see here that the names and symbols of the tags used, along with the designation that all have an "Antibody Capture" type.
+One you might note looks different is the `IgG1` control, which is actually a mouse antibody used as a negative control. 
+
 
 ### Clustering redux
 
-These data don't have clusters assigned yet, so let's add those, using the PCA matrix we have for the gene expression data.
+While these data did have dimension reduction performed, they were not analyzed for clustering yet.
+
+Let's assign some clusters to our cells, using graph-based clustering and default parameters, taking as input the PCA matrix that was previously calculated.
+Note that this PCA matrix and the UMAP built from it were derived from the gene expression data, so the clustering is going to reflect the gene expression data only.
+While we have the ADT data, it is _not_ being used for this stage of the analysis.
 
 
 ```{r}
 # perform clustering
-nn_k <- 20
 nn_clusters <- bluster::clusterRows(
+  # PCA input
   reducedDim(sce, "PCA"), 
-  bluster::NNGraphParam(
-    k = nn_k, 
-    type = "jaccard",
-    cluster.fun = "louvain"
-  )
+  # graph clustering & parameters
+  bluster::NNGraphParam()
 )
 
 # add clusters to colData
 sce$nn_cluster <- nn_clusters
 ```
 
-```{r}
-scater::plotUMAP(sce, 
-                 colour_by = "nn_cluster")
+Now we can plot the clusters we have identified with our old friend `scater::plotUMAP()`.
+
+```{r plot clusters}
+# plot clusters
+scater::plotUMAP(sce, colour_by = "nn_cluster") + 
+  # rename the legend
+  guides(colour = guide_legend(title = "Cluster"))
 ```
-But what are these clusters? Do they correspond to cell types?
+But what are these clusters, really? 
+Do they correspond to particular cell types that we are interested in?
 
-## Cell surface markers
+Does it bother you that we just used the default nearest-neighbor graph clustering parameters?
+Do you know what those were?
 
-The markers we have were presumably chosen for the fact that they mark particular cell types, so let's look at a few.
 
-CD3 is a marker of T-cells 
+## Investigating cell types
 
-```{r}
-scater::plotUMAP(sce, 
-                 colour_by = "CD3")
+### Using cell surface marker data
+
+The first way we will work on to identify the cell types of the individual cells in our sample is to work with the ADT markers that are part of this data set. 
+These antibody markers were (hopefully) chosen for their relevance to the cell population we are working with.
+
+The first marker we will look at is `CD3`, which is a protein complex that is found on the surface of T cells.
+We can plot where this is found using the same `plotUMAP()` function as before. 
```

Comment:
We're not plotting  _presence_ ("where this is found") of CD3 but rather thwe _amount of_ CD3. 

suggested rephrase - 
```suggestion
We can again use the `plotUMAP()` function to color cells by `CD3` expression levels. 
```